src/main/java/com/onaio/steps/adapters/ParticipantAdapter.java

```diff
@@ -67,13 +68,15 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
     private void setTextInView(View view, Participant participantAtPosition) {
         TextView participantPidView = (TextView) view.findViewById(R.id.main_text);
+        ImageView cloudIcon = (ImageView) view.findViewById(R.id.cloud_icon);
```

Comment:
Place the `cloud_icon` in the `drawable` folder since we only have one version and not density-specific copies